python/paddle/fluid/contrib/slim/tests/slim_int8_mkldnn_post_training_quantization.md

```diff
@@ -0,0 +1,61 @@
+# SLIM Post-training quantization (INT8 MKL-DNN)
+
+This document describes how to use [Paddle Slim](https://github.com/PaddlePaddle/FluidDoc/blob/develop/doc/fluid/advanced_usage/paddle_slim/paddle_slim.md) to convert a FP32 ProgramDesc with FP32 weights to an INT8 ProgramDesc with FP32 weights on GoogleNet, MobileNet-V1, MobileNet-V2, ResNet-101, ResNet-50, VGG16 and VGG19. Paddle Slim MKL-DNN post-training quantization strategy accuracy results on above 7 models are as follows:
+
+## Accuracy benchmark
+
+>**I. Top-1 Accuracy on Intel(R) Xeon(R) Gold 6271**
+
+| Model        | Dataset                        | FP32 Accuracy   | INT8 Accuracy   | Accuracy Diff   |
```

Comment:
Could you write something like "Dataset: ILSVRC2012 Validation dataset" before the table? 
No need a dataset column, since it is the same for 7 models. 

same in `int8_mkldnn_quantization.md`.